src/main/kotlin/apply/ui/admin/mission/MissionForm.kt

```diff
@@ -20,26 +23,51 @@ class MissionForm() : BindingIdentityFormLayout<MissionData>(MissionData::class)
     }
     private val startDateTime: DateTimePicker = DateTimePicker("시작 일시")
     private val endDateTime: DateTimePicker = DateTimePicker("종료 일시")
+
+    private var judgeItemId: Long = 0L
+    private val testName: TextField? = TextField("채점에 사용할 미션 이름")
+    private val programmingLanguage: Select<ProgrammingLanguage>? =
+        createItemSelect<ProgrammingLanguage>("언어 선택").apply {
+            setItems(*ProgrammingLanguage.values())
+            setItemLabelGenerator { it.language }
+        }
+    private val evaluationItem: Select<EvaluationItemData>? = createItemSelect<EvaluationItemData>("평가 항목").apply {
+        setItemLabelGenerator(EvaluationItemData::title)
+        isEmptySelectionAllowed = false
+    }
+
     private val submittable: RadioButtonGroup<Boolean> = createBooleanRadioButtonGroup("제출 여부", "제출 시작", "제출 중지", false)
     private val hidden: RadioButtonGroup<Boolean> = createBooleanRadioButtonGroup("공개 여부", "비공개", "공개", true)
 
     init {
-        add(title, evaluation, startDateTime, endDateTime, description, submittable, hidden)
+        add(
+            title, evaluation, startDateTime, endDateTime, description,
+            Hr(), testName, programmingLanguage, evaluationItem, Hr(),
+            submittable, hidden
+        )
         setResponsiveSteps(ResponsiveStep("0", 1))
         drawRequired()
     }
 
     constructor(
-        evaluations: List<EvaluationSelectData>
+        evaluations: List<EvaluationSelectData>,
+        listener: (id: Long) -> List<EvaluationItemData>,
     ) : this() {
         evaluation.setItems(evaluations)
+        evaluation.addValueChangeListener {
+            val evaluationItems: List<EvaluationItemData> = mutableListOf()
+            evaluationItem?.setItems(evaluationItems.plus(listener(it.value.id)))
+        }
     }
 
     override fun bindOrNull(): MissionData? {
-        return bindDefaultOrNull()
+        val missionData = bindDefaultOrNull()
+        missionData?.judgeItemId = judgeItemId
+        return missionData
```

Comment:
```suggestion
        return bindDefaultOrNull().apply {
          this?.judgeItemId = judgeItemId
        }
```
a : 이렇게도 가능할 수도 있는데 가독성 측면에서는 잘 모르겠네요 🧚‍♀️